libs/interpret-text/src/lib/TextExplanationDashboard/Control/TextHighlighting/TextHightlighting.tsx

```diff
@@ -28,51 +36,55 @@ export class TextHighlighting extends React.PureComponent<IChartProps> {
     const k = this.props.topK;
     const sortedList = Utils.sortedTopK(importances, k, this.props.radio);
     return (
-      <Stack
-        id="TextHighlighting"
-        horizontal
-        horizontalAlign="start"
-        tokens={textStackTokens}
-        wrap
-        styles={textStackStyles}
-      >
-        {text.map((word, wordIndex) => {
-          let styleType = classNames.normal;
-          const score = importances[wordIndex];
-          let isBold = false;
-          if (sortedList.includes(wordIndex)) {
-            if (score > 0) {
-              styleType = classNames.highlighted;
-            } else if (score < 0) {
-              styleType = classNames.boldunderline;
-              isBold = true;
-            } else {
-              styleType = classNames.normal;
-            }
-          }
-          if (isBold) {
-            return (
-              <Label
-                key={wordIndex}
-                className={styleType}
-                title={score.toString()}
-              >
-                {word}
-              </Label>
-            );
-          }
+      <Stack styles={scrollablePaneStyles}>
+        <ScrollablePane scrollbarVisibility={ScrollbarVisibility.auto}>
+          <Stack
+            id="TextHighlighting"
+            horizontal
+            horizontalAlign="start"
+            tokens={textStackTokens}
+            wrap
+            styles={textStackStyles}
+          >
```

Comment:
Does it work to only have one stack ? like <ScrollablePane> <Stack>...</Stack> </ScrollablePane>? I'm not sure what the outstack is adding that the pane couldn't

Reply:
the scrollable pane needs to go inside a stack with position: relative according to docs... but I can't do that in the inner stack and also if scroll goes in inner stack it messes up the spacing of the words for some reason